articles/basic-types-and-value-literals.html

```diff
@@ -159,11 +159,12 @@ <h3>Zero Values</h3>
 	different numeric types may have different sizes in memory.
 </li>
 <li>
-	The zero value of a string type is a blank string.
+	The zero value of a string type is a empty string like <code>""</code>.
 </li>
 </ul>
 </div>
 
+<p class="anchor" id="basic_value_literals"></p>

```

Comment:
I think `_` is better because using `-` in identifier name is not a good choice.
Maybe one day I exchanged all names defined with `-` to use `_`, but for now, I agree on maintaining consistency.

Reply:
Aha, `_` may be good for a Go identifier, but in html and css world, `-` is used more popularly.